corehq/apps/users/templates/users/partials/edit_role_modal.html

```diff
@@ -20,13 +20,14 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
           <div class="form-group">
             <label class="col-sm-4 control-label"
                    for="role-name">
-              {% trans "Role Name" %}
+              {% trans "Role Name*" %}
```

Comment:
Adding the asterisk is a nice touch. This is the nitpickiest of nitpicks, but can you wrap it in `<span class="asteriskField">`?

It's a super-nitpick because that class doesn't currently add any styles. But if we do want to style these asterisks, it's going to be ~impossible to find them, so it's good to mark them with a class.